android/lib/src/main/java/com/swmansion/gesturehandler/GestureHandlerOrchestrator.kt

```diff
@@ -305,7 +296,7 @@ class GestureHandlerOrchestrator(
       }
     }
 
-    event.setLocation(oldX, oldY)
+    event.recycle()
```

Comment:
are you sure it is safe to recycle event here? It seem like `transormEventToViewCoords` may sometimes create a new event that should be recycled and other times it may reuse the one provided to it. The latter case matches the previous behavior of this method and we never called recycle here. Not saying this is bad but just look suspicious.

Reply:
`transormEventToViewCoords` never creates a new event, it always transforms the one it receives but I can see how the code of the method might've suggested otherwise. To be hones I don't remember why I've done it like that (it might've been a result of some logic that used to be there), but I've simplified it in https://github.com/software-mansion/react-native-gesture-handler/pull/2156/commits/78c37cd5f7984ca0d7590d76c00aef7513a71574 so it should be much clearer that it doesn't create a new event.

That said, It should be safe to recycle there since `transformEventToViewCoords` receives (and returns transformed) a copy of the source event: https://github.com/software-mansion/react-native-gesture-handler/blob/78c37cd5f7984ca0d7590d76c00aef7513a71574/android/lib/src/main/java/com/swmansion/gesturehandler/GestureHandlerOrchestrator.kt#L255